### ISSUE TITLE
[bug] webhooks - request entity too large

### ISSUE THREAD

**Issue by morremeyer** [mend/renovate-ce-ee#354]
We're using renovate-ce with GitHub.

Since migrating to renovate-ce from renovate on-prem, ~all webhooks~ some of the webhooks are failing with the following error stack:

```json
{
  "err": {
    "message": "request entity too large",
    "expected": 133688,
    "length": 133688,
    "limit": 102400,
    "type": "entity.too.large",
    "stack": "PayloadTooLargeError: request entity too large\n    at readStream (/usr/src/app/node_modules/raw-body/index.js:156:17)\n    at getRawBody (/usr/src/app/node_modules/raw-body/index.js:109:12)\n    at read (/usr/src/app/node_modules/body-parser/lib/read.js:79:3)\n    at jsonParser (/usr/src/app/node_modules/body-parser/lib/types/json.js:135:5)\n    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/usr/src/app/node_modules/express/lib/router/index.js:328:13)\n    at /usr/src/app/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/usr/src/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/usr/src/app/node_modules/express/lib/router/index.js:280:10)\n    at urlencodedParser (/usr/src/app/node_modules/body-parser/lib/types/urlencoded.js:100:7)\n    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/usr/src/app/node_modules/express/lib/router/index.js:328:13)\n    at /usr/src/app/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/usr/src/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/usr/src/app/node_modules/express/lib/router/index.js:280:10)\n    at expressInit (/usr/src/app/node_modules/express/lib/middleware/init.js:40:5)\n    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/usr/src/app/node_modules/express/lib/router/index.js:328:13)\n    at /usr/src/app/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/usr/src/app/node_modules/express/lib/router/index.js:346:12)"
  }
}
```

We're currently running `ghcr.io/mend/renovate-ce:6.1.0-full`. 

It looks like the post body limit is at `1m`, while GitHub's webhooks have larger bodies than that.

**Comment by rarkins** [comment]
What's in the 1mb??

**Comment by morremeyer** [comment]
Looking at the failing webhooks, it seems to be affecting the Dependency Dashboard ones for our larger repositories.

These contain the whole comment for the issue, meaning that for repositories with a large number of dependencies, there's a lot of text.

Interpreting the `102400` as a byte size (not kb as initially mentioned, sorry), I get 102.4 kB as maximum size. For one of our repositories, the webhook body has 132 kB, so it errors.

**Comment by PhilipAbed** [comment]
might be because of express default, 
![image](https://github.com/mend/renovate-ce-ee/assets/42116482/ecff250d-7d84-419c-81f0-c12adae63525)

**Comment by rarkins** [comment]
@PhilipAbed @Gabriel-Ladzaretti isn't this solved in 6.1.1?

**Comment by PhilipAbed** [comment]
supposed to be merged already

**Comment by rarkins** [comment]
Fixed in 6.1.1
